### ISSUE TITLE
Add Multiple Multipart POST 

### ISSUE THREAD

**Issue by jmorille** [smartrics/RestFixture#197]
Add the complexe multipart request with this API

````
|Table: Rest Fixture | http://${SERVER}/api/rest/ | reqparam |
|addMultipartFile| ${FITNESSE_ROOTPATH}/FitNesseRoot/files/myfile1.pdf| fileParam1 |
|addMultipartFile| ${FITNESSE_ROOTPATH}/FitNesseRoot/files/myfile12.pdf| fileParam2 |  application/octet-stream |
|addMultipartString| {"firstname":"Test","lastname":"test" } | request | application/json |  
| POST |/myservice | 204 |||
````

This Pull request keep the backward compatibility and is divide in 2 Parts for the Two project

**Comment by jmorille** [comment]
two method was added for file and String
All parameter after "Name of File" or " String content " are optionnal
````
/**
	 * Allows setting of the the multi-part file to upload.
	 * <p>
	 * <code>| addMultipartFile | Name of file | Name of form parameter for the uploaded file | ContentType of file | CharSet of file |</code>
	 * <p>
	 * body text should be location of file which needs to be sent
	 */
	@SuppressWarnings({ "rawtypes", "unchecked" })
	public void addMultipartFile() {
````

````
/**
	 * Allows setting of the multi-part String.
	 * <p>
	 * <code>| addMultipartString | String content | Name of form parameter  | ContentType of String | CharSet of String |</code>
	 * <p>
	 * body text should be location of file which needs to be sent
	 */
	@SuppressWarnings({ "rawtypes", "unchecked" })
	public void addMultipartString() {
````

**Comment by jmorille** [comment]
For working need to maange the Pull request https://github.com/smartrics/RestClient/pull/9 in RestClient project before

**Comment by smartrics** [comment]
thank you for the contribution. i have merged and released version 4.2
also added a spec for [slim](https://github.com/smartrics/RestFixtureLiveDoc/tree/master/src/main/resources/FitNesseRoot/RestFixtureTests/MultipleMultipartPostTests) and [fit](https://github.com/smartrics/RestFixtureLiveDoc/tree/master/src/main/resources/FitNesseRoot/RestFixtureFitTests/MultipleMultipartPostTests)
